modules/raveboard/src/index.html

```diff
@@ -7,6 +7,9 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <!--
```

Comment:
Totally forgot about the font. We should download it and serve as an asset. Not a priority for now since the standard font looks OK, too.

Reply:
Hmm, yeah